website/docs/r/service_fabric_cluster.html.markdown

```diff
@@ -88,6 +88,10 @@ The following arguments are supported:
 
 * `upgrade_policy` - (optional) A `upgrade_policy` block as defined below.
 
+* `sf_zonal_upgrade_mode` - (Optional) Specifies the logical grouping of VMs in upgrade domains. Possible values are `Hierarchical` or `Parallel`.
```

Comment:
does sf stand for service fabric? if so that is implied by the resource name?
```suggestion
* `zonal_upgrade_mode` - (Optional) Specifies the logical grouping of VMs in upgrade domains. Possible values are `Hierarchical` or `Parallel`.
```

Reply:
@katbyte - I guess it is to differentiate from the other parameter `vmss_zonal_upgrade_mode` (I just copied what the API uses)

Reply:
yea that makes sense, thou i imagine without any prefix it is implied - could we either go with zonal_upgrade_mode or servicefabric_zonal_upgrade_mode to prevent the ambiguity of sf?